core/fsutils/fsutils.go

```diff
@@ -43,7 +44,7 @@ func (fs *FS) CopyFile(filePath, targetPath string) error {
 func (fs *FS) Execute(op *Operation) (err error) {
 	dirPath := fs.root
 	if op.SourceDir != "" {
-		dirPath = filepath.Join(fs.root, op.SourceDir)
+		dirPath = fs.normalizePath(op.SourceDir)
```

Comment:
Why did we drop the root here?

Reply:
`normalizePath` does the work of prepending the `fs.root` if it's not already in the path.